resolwe_bio/processes/fusion_calling/arriba.py

```diff
@@ -0,0 +1,132 @@
+"""Gene fusion detection with Arriba."""
+
+from pathlib import Path
+from plumbum import TEE
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class Arriba(Process):
+    """Run Arriba for fusion detection from RNA-Seq data.
+
+    Arriba is a command-line tool for gene fusion detection from RNA-Seq data.
+    This process detects gene fusions from RNA-Seq data using the Arriba tool.
+    The input can be a BAM file from the STAR aligner, and additional optional
+    inputs such as blacklist, protein domains, and known fusions files can be provided.
+    More information about Arriba can be found in the
+    [Arriba manual](https://arriba.readthedocs.io/en/latest/) and in
+    the [original paper](https://genome.cshlp.org/content/31/3/448)
+    The current version of Arriba is 2.4.0.
+    """
+
+    slug = "arriba"
+    name = "Arriba"
+    process_type = "data:genefusions:arriba"
+    version = "1.0.0"
+    category = "RNA-Seq"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {"type": "sample"}
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/rnaseq:6.2.0"}
+        },
+        "resources": {
```

Comment:
Reduce the processing requirements to 1 thread and 12288 memory (from docs: "Arriba usually consumes less than 10 GB of RAM"). We'll be able to test this limits and potentially increase later, as needed.